db/column_family.cc

```diff
@@ -391,6 +391,11 @@ void SuperVersion::Init(MemTable* new_mem, MemTableListVersion* new_imm,
   refs.store(1, std::memory_order_relaxed);
 }
 
+SequenceNumber SuperVersion::GetSmallestMemSeqno() const {
```

Comment:
Nit: I think this method should be named `GetEarliestMemTableSequenceNumber` for consistency with the naming of similar methods elsewhere. Also, this looks very similar to `DBImpl::GetEarliestMemTableSequenceNumber`.

PS I wouldn't mind if all of the uses of `earliest` where switched to `smallest` for consistency with `FileMetadata::{smallest,largest}_seqno`, but I'll leave it to the FB RocksDB folks to decide if they want to do that.